packages/messaging/src/controllers/window-controller.ts

```diff
@@ -1,6 +1,6 @@
 /**
  * @license
- * Copyright 2017 Google Inc.
+ * Copyright 2019 Google Inc.
```

Comment:
Again with the copyright

I'll stop mentioning it but there are other files too :stuck_out_tongue: